### DISCUSSION TITLE
Add (fake) language-agnostic redirect in ReadTheDocs configuration

### DISCUSSION THREAD

**Discussion by jdknight** [sphinx-doc/sphinx#13014] [Feature requests and ideas]
It may be nice to support language-agnostic URLs for documentation. For example, instead of linking of the page:

    https://www.sphinx-doc.org/en/master/usage/installation.html

A user could use a link such as:

    https://www.sphinx-doc.org/usage/installation.html

Read the Docs does not (yet?) support automatic language-agnostic URLs on projects, but individual projects can provide a superficial support language-agnostic URLs by adding a redirect to their default language. If such a redirect were to be added, it could allow other sites/documents reference Sphinx documentation without needing to explicit add the `/en/master` in the URL. And then maybe sometime in the future if Read the Docs can support some level of automatic language-agnostic URLs support, existing third-party resources that use language-agnostic links will have a better user experience for non-English users.

Here is an example Read the Docs redirect configuration (see an example: [1] v. [2]):

![image](https://github.com/user-attachments/assets/536715ef-83ae-4d8b-8bc0-2da7310ab8bb)

No worries if not desired though, considering:
- I am not aware of Read the Docs plan to support such a feature.
- I have not yet created a discussion or push for such a capability in Read the Docs's community.

[1]: https://sphinxcontrib-confluencebuilder.readthedocs.io/configuration/
[2]: https://sphinxcontrib-confluencebuilder.readthedocs.io/en/stable/configuration/

**Comment by jdillard** [comment]
How would a user get such a link as `https://www.sphinx-doc.org/usage/installation.html`? Seems like adding a lot of complexity for a hidden feature that would rarely be used.

**Reply by jdknight** [reply]
Language-agnostic URLs could be manually crafted and used in other documentation referencing content like Sphinx.

From what I can tell, this technique is used in a some of Read the Docs-used projects. Some random ones picked out are:

- Flask: https://flask.palletsprojects.com/deploying/ -> https://flask.palletsprojects.com/en/stable/deploying/
- Python Packaging User Guide: https://packaging.python.org/tutorials/ -> https://packaging.python.org/en/latest/tutorials/

Granted, I have must just stumbled upon the only two that support this capability.
